extensions/tmtv/src/init.js

```diff
@@ -0,0 +1,42 @@
+import measurementServiceMappingsFactory from './utils/measurementServiceMappings/measurementServiceMappingsFactory';
+
+const CORNERSTONE_3D_TOOLS_SOURCE_NAME = 'Cornerstone3DTools';
+const CORNERSTONE_3D_TOOLS_SOURCE_VERSION = '0.1';
+/**
+ *
+ * @param {Object} servicesManager
+ * @param {Object} configuration
+ * @param {Object|Array} configuration.csToolsConfig
+ */
+export default function init({
+  servicesManager,
+  commandsManager,
+  configuration,
+}) {
+  const {
+    MeasurementService,
+    DisplaySetService,
+    Cornerstone3DViewportService,
+  } = servicesManager.services;
+
+  const { RectangleROIStartEndThreshold } = measurementServiceMappingsFactory(
```

Comment:
So the mapping is added here, but the tool is added in the cornerstone extension? Since they are for the TMTV mode, they should both really be added in this extension?